libra-swarm/src/main.rs

```diff
@@ -8,36 +8,32 @@ use structopt::StructOpt;
 use tools::tempdir::TempPath;
 
 #[derive(Debug, StructOpt)]
-#[structopt(
-    name = "libra_swarm",
-    author = "Libra",
-    about = "Libra swarm to start local nodes"
-)]
+#[structopt(about = "Libra swarm to start local nodes")]
 struct Args {
     /// Number of nodes to start (1 by default)
-    #[structopt(short = "n", long = "num_nodes")]
-    pub num_nodes: Option<usize>,
+    #[structopt(short = "n", long, default_value = "1")]
+    pub num_nodes: usize,
     /// Enable logging
-    #[structopt(short = "l", long = "enable_logging")]
+    #[structopt(short = "l", long)]
```

Comment:
Would comments on the default behavior make sense here?

Reply:
yep will do